plugins/kettle-dummy-plugin/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.pentaho.kettle.plugins</groupId>
+    <artifactId>plugins</artifactId>
+    <version>8.0-SNAPSHOT</version>
+  </parent>
+  <groupId>org.pentaho.kettle.plugins</groupId>
+  <artifactId>kettle-dummy-plugin-parent</artifactId>
+  <version>8.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <name>${project.artifactId}</name>
+  <description>This plugin is for pentaho Data integration (ETL) a.k.a kettle</description>
+  <modules>
+    <module>core</module>
+    <module>assemblies</module>
+  </modules>
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+      <comments>A business-friendly OSS license</comments>
+    </license>
+  </licenses>
+  <properties>
+    <org.eclipse.swt.version>4.6</org.eclipse.swt.version>
+    <jface.version>3.3.0-I20070606-0010</jface.version>
+    <junit.version>4.7</junit.version>
+    <pentaho-kettle.version>${project.version}</pentaho-kettle.version>
+  </properties>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>pentaho-kettle</groupId>
+        <artifactId>kettle-engine</artifactId>
+        <version>${project.version}</version>
+        <scope>provided</scope>
+      </dependency>
+      <dependency>
+        <groupId>pentaho-kettle</groupId>
+        <artifactId>kettle-core</artifactId>
+        <version>${project.version}</version>
```

Comment:
don't use `${project.version}` here, use instead the one you defined `pentaho-kettle.version`